tvb_framework/tvb/core/entities/storage/datatype_dao.py

```diff
@@ -289,20 +289,20 @@ def get_data_in_project(self, project_id, visibility_filter=None, filter_value=N
         resulted_data = []
```

Comment:
Could it be useful to have a unit-test for `dao.get_data_in_project `?
Then that unit-test should fail on the current master when `visibility_filter` param is given, but pass otherwise - and be fixed in this branch.